epub33/rs/index.html

```diff
@@ -1419,6 +1419,14 @@ <h3>Navigation document processing</h3>
 							data-cite="epub-33#confreq-nav-a">links and headings</a> in the <a
 							data-cite="epub-33#sec-nav-toc"><code>toc nav</code> element</a> [[epub-33]].</p>
 				</li>
+				<li>
+					<p id="confreq-nav-alt-text">MUST, when generating non-HTML based navigation widgets, replace
```

Comment:
```suggestion
					<p id="confreq-nav-alt-text" data-tests="#nav-non-text_img,#nav-non-text_img_title">MUST, when generating non-HTML based navigation widgets, replace
```

Reply:
The tests have been approved, I will merge this comment that is bound to the tests.